sdk/storage/Azure.Storage.DataMovement.Blobs/src/DataMovementBlobConstants.cs

```diff
@@ -7,12 +7,20 @@ namespace Azure.Storage.DataMovement.Blobs
 {
     internal class DataMovementBlobConstants
     {
+        internal class ResourceId
+        {
+            internal const string BlockBlob = "BlockBlob";
+            internal const string PageBlob = "PageBlob";
+            internal const string AppendBlob = "AppendBlob";
+        }
+
         internal class SourceCheckpointData
         {
             internal const int SchemaVersion = 1;
```

Comment:
Will need to bump the source/destination checkpoint schema versions.

Reply:
Made an issue for it.